packages/claim-cli/src/utils/network-params.ts

```diff
@@ -0,0 +1,23 @@
+export interface NetworkParams {
+	api: string;
+	rpc: string;
+	l2Claim: string;
+	maxFeePerGas: bigint;
+	maxPriorityFeePerGas: bigint;
+}
+
+export const Mainnet = {
+	api: 'https://token-claim-api.lisk.com/rpc',
+	rpc: 'https://rpc.api.lisk.com',
+	l2Claim: '0xD7BE2Fd98BfD64c1dfCf6c013fC593eF09219994',
+	maxFeePerGas: BigInt(1002060),
+	maxPriorityFeePerGas: BigInt(1000000),
```

Comment:
where do these come from?

Reply:
This is the data obtained from `provider.getFeeData()` during implementation. It will be used only when the FeeData in unlikely event returns `null`. Since the type of `getFeeData` is `null | bigint`, this value is simply a fallback.

According to EIP1559: `maxFeePerGas = maxPriorityFeePerGas + baseFee * integer-n`. n increases and caps at `maxFeePerGas`. And currently in Lisk L2 baseFee is 1030 wei.

i.e. `1002060 wei = 1000000 wei + 2 * 1030 wei`

[Example Txn](https://blockscout.lisk.com/tx/0x1bb069ad610054c5de7469b39d382ea945dd7395afaa5b92f2496c0d000d603b)